homeassistant/components/zone/__init__.py

```diff
@@ -329,7 +328,7 @@ def should_poll(self) -> bool:
         """Zone does not poll."""
         return False
 
-    async def async_update_config(self, config: dict) -> None:
+    async def async_update_config(self, config: ConfigType) -> None:
```

Comment:
Can `__init__` and `from_yaml` be adjusted to use `ConfigType`, to match in `zone`?